x-pack/plugins/cases/server/services/cases/transform.ts

```diff
@@ -189,6 +190,35 @@ export function transformSavedObjectToExternalModel(
   };
 }
 
+export function transformResolveSavedObjectToExternalModel({
```

Comment:
This function is pretty close to `transformSavedObjectToExternalModel`. Do you think we could have it call `transformSavedObjectToExternalModel` instead of doing the `transformESConnectorOrUseDefault` `transformESExternalService` calls? or do we run into type problems?